build.rs

```diff
@@ -17,50 +17,58 @@ fn main() {
     if !feature!("SYSTEM") {
         let cblas = feature!("CBLAS");
         let lapacke = feature!("LAPACKE");
-        let output = PathBuf::from(variable!("OUT_DIR").replace(r"\", "/"));
-        let mut make = Command::new("make");
-        make.args(&["libs", "netlib", "shared"])
-            .arg(format!("BINARY={}", binary!()))
-            .arg(format!("{}_CBLAS=1", switch!(cblas)))
-            .arg(format!("{}_LAPACKE=1", switch!(lapacke)))
-            .arg(format!("-j{}", variable!("NUM_JOBS")));
-        let target = match env::var("OPENBLAS_TARGET") {
-            Ok(target) => {
-                make.arg(format!("TARGET={}", target));
-                target
+        if cfg!(target_os="windows") {
```

Comment:
`system` feature means "use system OpenBLAS" for UNIX systems, and "system OpenBLAS" is the vcpkg-managed one for Windows IMO. This part should be in `if feature("SYSTEM")` branch, and throw error here because non-`system` OpenBLAS is currently not available for Windows now.

Reply:
Yeah that makes sense. I will make this change.